exercises/pangram/pangram_test.py

```diff
@@ -42,14 +43,20 @@ def test_missing_letters_replaced_by_numbers(self):
 
     def test_pangram_with_mixedcase_and_punctuation(self):
         self.assertIs(
-            is_pangram('"Five quacking Zephyrs jolt my wax bed."'),
+            is_pangram('\"Five quacking Zephyrs jolt my wax bed.\"'),
```

Comment:
The `\"` construct is for escaping the quote in the `json` - I don't think that they are needed here.